client/src/app/main-page/main-page.component.ts

```diff
@@ -0,0 +1,22 @@
+import { Component, OnInit } from '@angular/core';
+
+import { SpinnerService } from '../global-components/services/spinner.service';
+
+@Component({
+  selector: 'app-main-page',
+  templateUrl: './main-page.component.html',
+  styleUrls: ['./main-page.component.scss']
+})
+
+export class MainPageComponent implements OnInit {
+  isSpinnerVisible: boolean = true;
+  constructor(
+    private spinnerService: SpinnerService,
+  ) { }
+
+  ngOnInit(): void {
+    this.spinnerService.getIsSpinnerVisible$().subscribe((value: boolean) => {
```

Comment:
If you want to have a subscription pls do unsubscribe on component destroy. But it's better to use async pipe instead. isSpinnerVisible$ = this.spinnerService.getIsSpinnerVisible$();

Reply:
I use @ngneat/until-destroy